vcbuild.bat

```diff
@@ -297,8 +297,8 @@ where /R . /T *.gyp? >> .gyp_configure_stamp
 if defined nobuild goto sign
 
 @rem Build the sln with msbuild.
-set "msbcpu=/m:2"
-if "%NUMBER_OF_PROCESSORS%"=="1" set "msbcpu=/m:1"
+set "msbcpu=/maxcpucount"
+if defined NUMBER_OF_PROCESSORS set "msbcpu=%msbcpu%:%NUMBER_OF_PROCESSORS%"
```

Comment:
This will exhaust the memory available in some machines (including our CI). Ref https://github.com/nodejs/node/commit/6cfedf0481c3a35b6a1d501104288f31a0e5445c and https://github.com/nodejs/node/pull/12184#issuecomment-291622426 . I tested this extensively when I made this change, to find the best combination that uses a reasonable number of threads, and this was the best. CL will use one thread for each processor, but since some of our projects are quite small this helps a lot. If we did it the other way around, not limiting here and limiting in CL, V8 would keep compiling long after everything else finished.

Reply:
On my 4 CPU box this does not affect compilation time.

Reply:
I'll continue testing this, but meanwhile I'll remove this from this PR.